templates/scenarios/csharp/command-and-response/teamsfx/app.local.yml.tpl

```diff
@@ -26,18 +26,18 @@ provision:
     with:
       botId: ${{BOT_ID}}
       name: {%appName%}
-      messagingEndpoint: ${{TUNNEL_ENDPOINT}}/api/messages
+      messagingEndpoint: ${{BOT_ENDPOINT}}/api/messages
       description: ""
 
 configureApp:
   - uses: teamsApp/validate
     env: 
-      BOT_DOMAIN: ${{TUNNEL_DOMAIN}}
+      BOT_DOMAIN: ${{BOT_DOMAIN}}
```

Comment:
I believe this can be removed since `BOT_DOMAIN` is already loaded from *.env.local*.

Reply:
Thanks for the diea, done.